src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala

```diff
@@ -496,9 +496,32 @@ object SchemaTools extends SparkSessionWrapper {
                 isDebug,
                 newPrefix
               ).map(validateSchema(_, newPrefix))
+//              println("1st Item",validatedChildren(0))
+//              println("validatedChildren1 is ",struct(validatedChildren1.map(_.column): _*).alias(fieldStructure.name))
+//              println("fieldStructure.name is ",fieldStructure.name)
+//              println("dtArray.elementType is ",dtArray.elementType)
+
+//              println("validatedChildren is ",struct(validatedChildren.map(_.column): _*).alias(fieldStructure.name))
```

Comment:
cleanup all these old commented out lines.

Reply:
same for whole file.

Reply:
Done